app/services/subcase-is-approved.js

```diff
@@ -0,0 +1,30 @@
+import Service, { inject as service } from '@ember/service';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class SubcaseIsApprovedService extends Service {
+  @service store;
+
+  async approved(subcase) {
```

Comment:
Can you change the name to something that clarifies that it will return a boolean?  `isApproved`?